src/tag.rs

```diff
@@ -769,4 +769,19 @@ mod tests {
         let lyrics = tag.synchronised_lyrics().next().unwrap();
         assert_eq!(lyrics.description, "Description");
     }
+
+    #[test]
+    fn test_multiple_tags() {
+        // Create temp file
+        let tmp = tempfile::NamedTempFile::new().unwrap();
+        fs::copy("testdata/multi-tags.mp3", &tmp).unwrap();
```

Comment:
Copying test data is only needed if we write to it. This test only reads, so could you just read from the testdata directly?